src/keystore/serialize/mod.rs

```diff
@@ -61,11 +66,18 @@ impl KeyFile {
     /// * `dir` - path to destination directory
     /// * `addr` - a public address (optional)
     ///
-    pub fn flush<P: AsRef<Path>>(&self, dir: P, addr: Option<Address>) -> Result<(), Error> {
+    pub fn flush<P: AsRef<Path>>(&self,
+                                 dir: P,
+                                 addr: Option<Address>,
+                                 name: Option<String>,
+                                 desc: Option<String>)
```

Comment:
I think it totally wrong to send `name` and `desc` here, they should be already in self.